### DISCUSSION TITLE
Texture placement in subfolder unsupported by model-viewer editor?

### DISCUSSION THREAD

**Discussion by joaovbs96** [google/model-viewer#4440] [Q&A]
Hi! I was testing a very minimal GLTF file in the model-viewer editor ([test_sphere_with_subfolder.zip](https://github.com/google/model-viewer/files/12497997/test_sphere_with_subfolder.zip)), and I came across validation errors about "IMAGE_UNRECOGNIZED_FORMAT". However, my only texture is a PNG file.

After some fiddling around, I found out that the issue is that the texture is placed in a subfolder instead of directly alongside the GLTF file (here's another version without the subfolder: [test_sphere_no_subfolder.zip](https://github.com/google/model-viewer/files/12498013/test_sphere_no_subfolder.zip)). 

The model itself as well the texture paths within the GLTF file seem to be fine, and both versions of the model load correctly in other viewers (e.g. Babylon.JS, Khronos' Sample Viewer, [glTF Viewer](https://gltf-viewer.donmccurdy.com/)), without any validation errors. I believe this is overall also allowed by the GLTF specification, so it shouldn't be a validation warning.

Is the placement of textures within subfolders unsupported by model-viewer? I haven't seen this documented anywhere, though I might have missed it. Thanks in advance!

**Comment by elalish** [comment]
It's not about model-viewer, but our editor specifically. I think you're right that it's a limitation currently. Probably not a hard fix, but also not high priority. PRs welcome!
